app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -1199,7 +1199,7 @@ def test_update_experiment_branches_with_feature_configs_error(self):
         reference_branch = {"name": "control", "description": "a control", "ratio": 1}
         treatment_branches = [{"name": "treatment1", "description": "desc1", "ratio": 1}]
         invalid_feature_config_id = (
-            NimbusFeatureConfig.objects.all().order_by("-id").first().id + 1
+            NimbusFeatureConfig.objects.all().order_by("-id").all()[0].id + 1
```

Comment:
is there any reason to change first from all()[0]

Reply:
Yeah it's probably because `.first()` is `T | None` and you can't add +1 to None whereas `.all()[0]` is just `T`